src/Controls/src/Core/Entry/Entry.cs

```diff
@@ -179,15 +179,6 @@ public bool FontAutoScalingEnabled
 			set => SetValue(FontAutoScalingEnabledProperty, value);
 		}
 
-		/// <summary>
-		/// Determines whether text prediction and automatic text correction is enabled. Default value is <see langword="true"/>.
-		/// </summary>
-		public bool IsTextPredictionEnabled
```

Comment:
@mattleibow @mandel-macaque is this a binary break? 
The PR moves this property up to the base class.

Reply:
As long as the base class is in the same binary, this is not a breaking change, otherwise, it is a binary break.

Reply:
src/Compatibility/Material/src/Tizen/Compatibility.Material.Tizen.csproj  references these classes so yeah ... its a break. I'll get on to fixing it

Reply:
Seems we're on the clear! Everything is inside controls :D